examples/docker-entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+if [ $KEYCLOAK_USER ] && [ $KEYCLOAK_PASSWORD ]; then
+    /opt/jboss/keycloak-demo/keycloak/bin/add-user.sh -u $KEYCLOAK_USER -p $KEYCLOAK_PASSWORD --silent
```

Comment:
This is adding a WildFly console user, not Keycloak admin user. Those are two different things. So please remove this line and only keep the add-user-keycloak